packages/core/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -274,3 +272,11 @@ export interface DropdownComponentProps extends CustomMenuBaseProps, CustomOptio
 }
 
 export type DropdownProps = DropdownComponentProps;
+
+export type DropdownChipColors = "PRIMARY" | "NEGATIVE" | "POSITIVE";
```

Comment:
Can we change these to lowercase?

Reply:
Yes, I will change it when I deprecate the chips enums